tests/common/snappi_tests/traffic_generation.py

```diff
@@ -570,7 +572,7 @@ def verify_in_flight_buffer_pkts(duthost,
     data_flow_config = snappi_extra_params.traffic_flow_config.data_flow_config
     tx_frames_total = sum(metric.frames_tx for metric in flow_metrics if data_flow_config["flow_name"] in metric.name)
     tx_bytes_total = tx_frames_total * data_flow_config["flow_pkt_size"]
-    dut_buffer_size = get_lossless_buffer_size(host_ans=duthost)
+    dut_buffer_size = get_lossless_buffer_size(host_ans=ingress_duthost)
```

Comment:
I believe this change is for cisco platform only? Also what if egress port is long link?

Reply:
@sdszhang , I am not sure about being cisco specific.
The reverse direction(short -> long) the buffer size is the incoming side and will be 64 MB only.

Reply:
just checked in the community meeting, this is also applicable for Nokia platform too.